experimental/db-connection/main.go

```diff
@@ -20,9 +20,11 @@ import (
 	"context"
```

Comment:
I think we can remove this file from this PR. It's unrelated to the v1alpha1 changes

Reply:
True, my bad!

Reply:
I see that this one is here still. That is why I think you have a unnecessary stuff. Can you remove it and run update-deps.sh again?